stories/exercises/06_exercise.mdx

```diff
@@ -0,0 +1,62 @@
+# 06. Customize components using settings.
+
+In the last exercise, we familiarized ourselves with the USWDS theme settings. In this section, we'll take that a step further by using component specific settings.
+
+Each USWDS component with settings will have this section. It is a snapshot of what customizations are possible without needing to write custom styles.
+
+**Tip:** The [Component settings](https://designsystem.digital.gov/documentation/settings/#component-settings-2) section of the Settings page provides a comprehensive list of all available component settings, their default values, and what type of token or value they take!
+
+1. Look over the available banner theme settings on the "Banner settings" section of the [component page](https://designsystem.digital.gov/components/banner/).
+
```

Comment:
After moving the guidance around, this "each USWDS component with settings will have this section" line doesn't make sense. 

**question:** What do you think about moving line 5 or even 5 + 7 under line 9?

```suggestion
**Tip:** The [Component settings](https://designsystem.digital.gov/documentation/settings/#component-settings-2) section of the Settings page provides a comprehensive list of all available component settings, their default values, and what type of token or value they take!

1. Look over the available banner theme settings on the "Banner settings" section of the [component page](https://designsystem.digital.gov/components/banner/).

Each USWDS component with settings will have this section. It is a snapshot of what customizations are possible without needing to write custom styles.
```

Reply:
I think it works for now, but we can revisit.